src/request/payload/event.ts

```diff
@@ -1475,6 +1476,26 @@ export interface ThreadBroadcastMessageEvent extends SlackEvent<"message"> {
   channel_type: AnyChannelType;
 }
 
+// Tombstone is currently an undocumented message subtype.
```

Comment:
Can you remove this line? Indeed, the document does not mention it at all but the reason behind it would be this subtype won't be delivered as a message event payload. This could be used only as previous_message property for message_deleted events.

Reply:
That's a good point. In that case, would it make more sense to name the type `TombstoneMessageItem` as it is not an event and is a snowflake compared to the rest of the events?